sbin/shipper_utils/systemctl_wrapper.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/env python3
```

Comment:
Should we move all of the relevant starting/stopping of systemctl out of `INSTALL_SUBMITTY_HELPER.sh` to use this module?